src/main/java/com/webdev/cosmo/cosmobackend/service/user/UserService.java

```diff
@@ -0,0 +1,51 @@
+package com.webdev.cosmo.cosmobackend.service.user;
+
+import com.webdev.cosmo.cosmobackend.service.api.User;
+import com.webdev.cosmo.cosmobackend.service.common.mapper.UserMapper;
+import com.webdev.cosmo.cosmobackend.service.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.openapitools.model.UserModel;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
```

Comment:
Remove @Service and create this class with @Bean approach, You can see this approach in Przemek's PR / commit